src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
 body {
   margin: 0;
 }
+
+$num-days: 31;
```

Comment:
It's not necessary to define constant like `$num-days` and lets check to the rest of them.